source/ptreferential/ptreferential.cpp

```diff
@@ -182,6 +187,15 @@ std::vector<idx_t> get_indexes(Filter filter,  Type_e requested_type, const Data
                 }
             }
         }
+    } else if(! filter.method.empty()) {
+        if (filter.object == "vehicle_journey" && filter.method == "has_headsign") {
+            for (const VehicleJourney* vj: d.pt_data->headsign_handler.get_vj_from_headsign(filter.value)) {
+                indexes.push_back(vj->idx);
+            }
+        } else {
+            throw parsing_error(parsing_error::partial_error,
```

Comment:
so, according to the code, only `vehicle_journey` can have method `has_headsign`, right? is that true?